pallets/loans/src/migrations/v1.rs

```diff
@@ -16,7 +16,7 @@ mod v0 {
 
 	#[derive(Encode, Decode, Clone, PartialEq, Eq, TypeInfo, RuntimeDebug, MaxEncodedLen)]
 	pub struct WriteOffState<Rate> {
-		pub overdue_days: u32,
+		pub overdue_secs: u32,
```

Comment:
Now we are in the second's magnitude, I would set the type to `Moment` that matches with what `maturity_date` is.

Reply:
If we do that, I think we'll need to write a migration, since Moment is a u64 rather than a u32. But beyond that, a time difference of u32 can afaik represent 4294967295 seconds = 49710 days, which I think suffices.

Reply:
But for now, we are avoiding migrations because it's not used, isn't it? (i.e. we are avoiding migrations in the refactor v3).

Yeah, in terms of precision, we're safe, although I think it seems weird from a reader's perspective to have seconds represented in the same pallet as `u64` and `u32` (Actually, this should be generalized and let the runtime choose the precision: #1277).

Reply:
Also curious if a migration is needed to go from `u32` to `u64`. Maybe because no information is lost, it's not an issue 🤔 . Do you know about this @wischli?